VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,47 @@
+# Video 4: AsyncAPI and OpenAPI
+
+Hello again! Welcome to a new video about AsyncAPI. This video will discuss what AsyncAPI and OpenAPI have in common and how they differ. 
+
+Ready or not, here we go! 
+
+## What is OpenAPI?
+
+OpenAPI is a standard for describing application programming interfaces (APIs). This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
+
+The current OpenAPI specification grew out of the predecessor project, Swagger. The development company SmartBear subjected the existing Swagger specification to an open license, handing over both maintenance and development of the initiative to OpenAPI. In addition to SmartBear, members of the OpenAPI initiative include industry giants such as Google, IBM, and Microsoft. The Linux Foundation also supports this project. 
+
+## Similarities between AsyncAPI and OpenAPI
+
+AsyncAPI's ultimate goal is to work with EDA architectures as simply as with Rest APIs. The AsyncAPI specification in the asynchronous world is analogous to OpenAPI in the synchronous world. 
+
+Both initiatives share numerous similarities:
+
+- Contract-first philosophy
+- Agnostic implementation
+- Similar syntax
+- A tooling ecosystem 
+- Extensible by including new properties with the prefix x-
+- They protect information consumption using different security schemes. (Example: user and password, API keys, certificates, OAuth, OIDC, etc.)
+
+
+## Differences between them
+
+ As we have seen, the two initiatives are significantly similar, but they also cover significantly different fields. REST APIs have a specific function that requires their methods and documentation approaches. REST APIs have, among others, a great solution in the form of OpenAPI, a set of standardized documentation methods for enabling tools and interactivity.
+ 
+ On the other hand, message-based APIs are different because they require solutions for the caveats and intrinsic considerations of message-based architectures. AsyncAPI's tagline says it best: "REST APIs have OpenAPI. Messaging has AsyncAPI."
```

Comment:
the focus has shifted from the type of the API to its style which could be a bit confusing. It should probably go like this,  "async APIs which are considered a message-driven API is different because they require solutions for..."
or
I hope this makes sense.